pkg/reconciler/openshift/tektonconfig/extension.go

```diff
@@ -65,10 +65,11 @@ func (oe openshiftExtension) PreReconcile(ctx context.Context, tc v1alpha1.Tekto
 	}
 
```

Comment:
should we add the param in TektonConfig CR with default value? similar to other defaults